sphinx_gallery/tests/tinybuild/doc/conf.py

```diff
@@ -73,7 +85,7 @@
     ],
     "backreferences_dir": "gen_modules/backreferences",
     "subsection_order": f"{util_root}.noop_key",
-    "within_subsection_order": "FileNameSortKey",
+    "within_subsection_order": CustomSortKey,
```

Comment:
somehow I missed this comment before, but I think it's now moot?  The string-aliases for builtin sorters are now comprehensively tested in `test_example_sorting` (as are FQN strings of custom funcs, and actual callable functions) so AFAICT it's actually preferable to keep this as-is (since in the parametrized test we only test callable functions, not callable classes).

Reply:
Yes totally agree, this is good as is.